pkg/plugin/v3/scaffolds/internal/templates/config/api/webhook.go

```diff
@@ -33,13 +33,19 @@ type Webhook struct { // nolint:maligned
 	file.BoilerplateMixin
 	file.ResourceMixin
 
-	// Is the Group domain for the Resource replacing '.' with '-'
-	GroupDomainWithDash string
-
+	// Version of webhook marker to scaffold
+	WebhookVersion string
 	// If scaffold the defaulting webhook
 	Defaulting bool
 	// If scaffold the validating webhook
 	Validating bool
+
+	// The below fields are always overridden.
+
+	// Is the Group domain for the Resource replacing '.' with '-'
+	GroupDomainWithDash string
+	// Marker argument string
+	WebhookVersionArg string
```

Comment:
nit:
```suggestion
	WebhookVersion string
```

Show not required the arg.